core/store/index.ts

```diff
@@ -202,6 +202,16 @@ export function initStore () {
       storeName: 'ordersHistory',
       driver: localForage[config.localForage.defaultDrivers['ordersHistory']]
     })),
+    cmsPagesCollection: new UniversalStorage(localForage.createInstance({
```

Comment:
this should be initialised in before/after module hooks and not bound to Vue instance